### DISCUSSION TITLE
Alternate install location

### DISCUSSION THREAD

**Discussion by kevin-harms** [intel/llvm#4868] [Ideas]
https://github.com/intel/llvm/blob/347b114579ed74f50afb3a3747db9955ba67f3c8/buildbot/configure.py#L104

Would it be possible to allow an alternate install location instead of in the obj-dir?

**Comment by alexbatashev** [comment]
@kevin-harms `--cmake-opt="-DCMAKE_INSTALL_PREFIX=/your/path"` should get you there.

**Reply by kevin-harms** [reply]
I can test this. It means passing CMAKE_INSTALL_PREFIX twice and assuming that last one takes precedence. I'd prefer to have an option to set it directly.
